MetricsDashboard.Core/Models/IDatabaseSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace MetricsDashboard.Core.Models
+{
+    public interface IDatabaseSettings
```

Comment:
This should be in "Configuration" folder

Reply:
moved to Core.Configuration